Created & Updated zarko_java_tasks package!

Zarez92 opened the pull request.

SINAN-ALBAYATI commented:
Hello Zarko
I hope you are doing well,,, do I need to take any action with #17 ??
it's  showing its deleted and I think it will be used in the future for
Taks ?

I apologize Zarko but I still confused about Git and Github

On Wed, 14 Jun 2023 at 17:53, Zarko Nikolic ***@***.***>
wrote:

> @Zarez92 <https://github.com/Zarez92> requested your review on: #17
> <https://github.com/Zarez92/JavaCodingTask-GroupHelene/pull/17> Created &
> Updated zarko_java_tasks package!.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Zarez92/JavaCodingTask-GroupHelene/pull/17#event-9533631259>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5T2GVZQCDJWWJ5N5QMPSFLXLIXFLANCNFSM6AAAAAAZG64PYY>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***
> com>
>

Zarez92 commented:
Hey Sinnan I know it looks little bit confusing,  but when you scroll down you can see all tasks. I deleted some classes,  added and changed but at the end it this is the final update from my branch. It's ok on our meeting we can go with git/gitHub I'll try to explain.

Sent from my T-Mobile 5G Device
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: SINAN-ALBAYATI ***@***.***>
Sent: Thursday, June 15, 2023 8:03:23 PM
To: Zarez92/JavaCodingTask-GroupHelene ***@***.***>
Cc: Zarko Nikolic ***@***.***>; Mention ***@***.***>
Subject: Re: [Zarez92/JavaCodingTask-GroupHelene] Created & Updated zarko_java_tasks package! (PR #17)


Hello Zarko
I hope you are doing well,,, do I need to take any action with #17 ??
it's showing its deleted and I think it will be used in the future for
Taks ?

I apologize Zarko but I still confused about Git and Github

On Wed, 14 Jun 2023 at 17:53, Zarko Nikolic ***@***.***>
wrote:

> @Zarez92 <https://github.com/Zarez92> requested your review on: #17
> <https://github.com/Zarez92/JavaCodingTask-GroupHelene/pull/17> Created &
> Updated zarko_java_tasks package!.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Zarez92/JavaCodingTask-GroupHelene/pull/17#event-9533631259>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5T2GVZQCDJWWJ5N5QMPSFLXLIXFLANCNFSM6AAAAAAZG64PYY>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***
> com>
>

—
Reply to this email directly, view it on GitHub<https://github.com/Zarez92/JavaCodingTask-GroupHelene/pull/17#issuecomment-1593862223>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A5SRWAQQE3XBRVH4KC7POQ3XLOPEXANCNFSM6AAAAAAZG64PYY>.
You are receiving this because you were mentioned.Message ID: ***@***.***>
